app/src/test/java/com/kickstarter/viewmodels/ProfileViewModelTest.java

```diff
@@ -210,38 +178,49 @@ public void testProfileViewModel_EmitsUserNameAndAvatar() {
       }
     };
 
-    final Environment env = environment().toBuilder().apiClient(apiClient).build();
-    final ProfileViewModel.ViewModel vm = new ProfileViewModel.ViewModel(env);
+    setUpEnvironment(environment().toBuilder().apiClient(apiClient).build());
+
+    this.avatarImageViewUrl.assertValues(user.avatar().medium());
+    this.userNameTextViewText.assertValues(user.name());
+  }
+
+  @Test
+  public void testProfileViewModel_MessagesButtonHidden() {
+    final Config config = ConfigFactory.config()
+      .toBuilder()
+      .features(Collections.singletonMap("android_messages", false))
```

Comment:
what about the case when `"android_messages": true` _and_ `"android_messages"` is absent? 🤔